tribe-ext-map-show-all-events-with-same-venue/src/tribe-events-ajax-maps.js

```diff
@@ -0,0 +1,704 @@
+
+			ts.params = $.param( ts.params );
+
+			/**
+			 * DEPRECATED: tribe_ev_collectParams has been deprecated in 4.0. Use collect-params.tribe instead
+			 */
+			$( te ).trigger( 'tribe_ev_collectParams' );
+			$( te ).trigger( 'collect-params.tribe' );
+
+		}
+
+		$( te ).on( 'reload-old-browser.tribe', function() {
+			tribe_generate_map_params();
+			window.location = td.cur_url + '?' + ts.params;
+		} );
+
+		/**
+		 * @function tribe_map_processOption
+		 * @desc tribe_map_processOption is the main ajax event query for map view.
+		 */
+
+		function tribe_map_processOption() {
+
+			if ( !ts.popping ) {
+				tribe_generate_map_params();
+				ts.pushstate = false;
+				if ( !ts.initial_load ) {
+					ts.do_string = true;
+				}
+			}
+
+			if ( invalid_date ) {
+				return;
+			}
+
+			$( '#tribe-events-content .tribe-events-loop' ).tribe_spin();
+			deleteMarkers();
+
+			$.post( GeoLoc.ajaxurl, ts.params, function( response ) {
+
+				/**
+				 * DEPRECATED: tribe_ev_ajaxStart and tribe_ev_mapView_AjaxStart have been deprecated in 4.0. Use ajax-start.tribe and map-view-ajax-start.tribe instead
+				 */
+				$( te ).trigger( 'tribe_ev_ajaxStart' ).trigger( 'tribe_ev_mapView_AjaxStart' );
+				$( te ).trigger( 'ajax-start.tribe' ).trigger( 'map-view-ajax-start.tribe' );
+
+				tf.enable_inputs( '#tribe_events_filters_form', 'input, select' );
+
+				if ( response.success ) {
+
+					ts.ajax_running = false;
+
+					td.ajax_response = {
+						'total_count': parseInt( response.total_count ),
+						'view'       : response.view,
+						'max_pages'  : response.max_pages,
+						'tribe_paged': response.tribe_paged,
+						'timestamp'  : new Date().getTime()
+					};
+
+					ts.initial_load = false;
+
+					var $the_content = $.parseHTML( response.html );
+
+					$( '#tribe-events-content' ).replaceWith( $the_content );
+
+					//If no events are returned, then hide Header
+					if ( response.total_count == 0 ) {
+						$( '#tribe-events-header' ).hide();
+					}
+
+					/**
+					 * Calls the filterMarkers function
+					 */
+
+					var marker_filter = response.markers;
+					var filteredTgMarkers = FilterMarkers( marker_filter );
+
+					$.each( filteredTgMarkers, function( i, e ) {
+						tf.map_add_marker( e.lat, e.lng, e.title, e.address );
+					} );
+
+					/**
+					 * @function FilterMarkers
+					 * @desc FilterMarkers filter by latitude and longitude and concatenate titles
+					 */
+
+					function FilterMarkers( tgMarkers ) {
+
+						var markers_list = [];
+						for ( var i = 0; i < tgMarkers.length; i++ ) {
+							var exist = false;
+							var existingMarker = tgMarkers[i];
+
+							markers_list.forEach( function( item ) {
+								if( existingMarker.lat == item.lat && existingMarker.lng == item.lng && existingMarker.title != item.title ) {
+									exist = true;
+									item.title = "<center>" + item.title + existingMarker.title.link( existingMarker.link ) + "<center>";
+								}
+							});
+
+							if( !exist ) {
```

Comment:
Here and a few other places, formatting should be:

`if ( ! condition )`

* Space before the parentheses (for control structures, function calls are different)
* Space before and after the bang/exclamation mark char `!`